app/assets/stylesheets/sidebar.scss

```diff
@@ -65,9 +65,8 @@
         text-align: center;
         position: relative;
         float: right;
-        margin-right: 25%;
         margin-top: 6px;
-        margin-left: auto;
+        margin-right: 50px;
```

Comment:
assuming this for the beta-tag, can you check different browsers and screen sizes?